src/test/java/org/embulk/input/zendesk/services/TestZendeskSupportAPIService.java

```diff
@@ -32,14 +39,17 @@
 
     private ZendeskSupportAPIService zendeskSupportAPIService;
 
+    private RecordImporter recordImporter;
+
     @Before
     public void prepare()
     {
         zendeskRestClient = mock(ZendeskRestClient.class);
+        recordImporter = mock(RecordImporter.class);
     }
 
     @Test
-    public void buildPathWithIncrementalForPreview()
+    public void testBuildPathWithIncrementalForPreview()
```

Comment:
There is no different between `Preview` and `Run` here: https://github.com/treasure-data/embulk-input-zendesk/blob/e3d40efac247026902f8471ea08560a08ec71121/src/test/java/org/embulk/input/zendesk/services/TestZendeskSupportAPIService.java#L83-L90 because it is skipped to check: https://github.com/treasure-data/embulk-input-zendesk/blob/e3d40efac247026902f8471ea08560a08ec71121/src/test/java/org/embulk/input/zendesk/services/TestZendeskSupportAPIService.java#L321

I think it is okay to keep one only.

Reply:
fixed

Reply:
Although it is not 100% related but I would like to spot few points here too:
1. https://github.com/treasure-data/embulk-input-zendesk/blob/273793bceaec3249a2cce9293ef08d693bf7de8f/src/test/java/org/embulk/input/zendesk/services/TestZendeskSupportAPIService.java#L59-L85, different between preview / run is still here
2. Because there is no different between `preview` and `run`, always mention it in test cases are redundant
3. `ZendeskSupportAPIService` only overrides `buildURI`, verify that method only seems enough to me, currently we're mocking the whole data and use its abstract: https://github.com/treasure-data/embulk-input-zendesk/blob/273793bceaec3249a2cce9293ef08d693bf7de8f/src/test/java/org/embulk/input/zendesk/services/TestZendeskSupportAPIService.java#L158, why don't you just follow the one you wrote in another class: https://github.com/treasure-data/embulk-input-zendesk/blob/273793bceaec3249a2cce9293ef08d693bf7de8f/src/test/java/org/embulk/input/zendesk/services/TestZendeskNPSService.java#L34-L42?